os/kernel/Kconfig

```diff
@@ -567,6 +567,17 @@ config SCHED_CPULOAD_TIMECONSTANT_LONG
 	default 10
 	depends on SCHED_MULTI_CPULOAD
 
+config SCHED_CPULOAD_SNAPSHOT
+	bool "Support snapshot for cpuload measurement in specific time period"
+	default y
+	---help---
+		If this option is selected, user can measure CPU load in the latest period of time.
```

Comment:
Need to be clarified here because it is a little hard to understand how it works.